### ISSUE TITLE
inspec 1.25.0 compliance_profile_name': undefined method `is_automate_server_pre_080' for Compliance::API:Class (NoMethodError)

### ISSUE THREAD

**Issue by timusmc96** [inspec/inspec#1840]
### Description

C:/opscode/chefdk/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.25.0/lib/bundles/inspec-compliance/target.rb:84:in `compliance_profile_name': undefined method `is_automate_server_pre_080' for Compliance::API:Class (NoMethodError)
Did you mean?  is_automate_server_pre_080?

### InSpec and Platform Version

1.25.0

### Replication Case
run inspec vendor --overwrite logged into chef automate

### Possible Solutions

If you have already ideas how to solve the issue, add them here.

### Stacktrace

```
/opt/chefdk/embedded/bin/inspec vendor --overwrite
/root/.chefdk/gem/ruby/2.3.0/gems/inspec-1.25.0/lib/bundles/inspec-compliance/target.rb:83:in `compliance_profile_name': undefined method `is_automate_server_pre_080' for Compliance::API:Class (NoMethodError)
Did you mean?  is_automate_server_pre_080?
        from /root/.chefdk/gem/ruby/2.3.0/gems/inspec-1.25.0/lib/bundles/inspec-compliance/target.rb:69:in `resolved_source
```

**Comment by cheeseplus** [comment]
Almost entirely sure it's just a missing `?` on https://github.com/chef/inspec/blob/3140eeae358ba8e1a0a026960453dcbdb156b4ca/lib/bundles/inspec-compliance/target.rb#L83

**Comment by adamleff** [comment]
@timusmc96 thanks so much for raising this issue, and we apologize for letting this one slip through. This is fixed in #1841 and released as InSpec v1.25.1. 

Thanks again, and have a great weekend.
